en/debug/eclipse_jlink.md

```diff
@@ -78,6 +78,16 @@ For more information, see: [https://gnu-mcu-eclipse.github.io/debug/jlink/instal
    ![Eclipse: GDB Segger Debug config: debugger tab](../../assets/debug/eclipse_settings_debug_config_gdb_segger_build_config_debugger_tab.png)
    ![Eclipse: GDB Segger Debug config: startup tab](../../assets/debug/eclipse_settings_debug_config_gdb_segger_build_config_startup_tab.png)
 
+## SEGGER Task-aware debugging
```

Comment:
@PetervdPerk-NXP 
Is this the same as "Thread aware debugging"? https://www.segger.com/products/debug-probes/j-link/tools/j-link-gdb-server/thread-aware-debugging/

Reply:
Yes it's I've added a heading to be more clear about it.